jaeger-core/src/test/java/io/jaegertracing/ConfigurationTest.java

```diff
@@ -328,6 +333,14 @@ public void testNoServiceName() {
     new Configuration(null);
   }
 
+  @Test
+  public void testOverrideServiceName() {
```

Comment:
Can you add a test using `withServiceName(null)`, and another with `withServiceName("")`? This should document what is expected to happen in such case.

Reply:
It's not strictly related to this PR, but I am happy to add it. 

Test for null service name is already there, I have added only for empty case.